fccodegen/_generator.py

```diff
@@ -1,26 +1,180 @@
-import fccodegen
+#
+# Functions related to generating model code.
+# TODO: Find a better name/description/layout
+#
+import fccodegen as cg
 import jinja2
 import os
+import sympy as sp
+import time
 
 
 def load_template(*name):
     """
     Returns a path to the given template
     """
-    path = os.path.join(fccodegen.DIR_TEMPLATE, *name)
+    path = os.path.join(cg.DIR_TEMPLATE, *name)
     #TODO: Check absolute path is still in DIR_TEMPLATE
     #TODO: Check exists
     #TODO: Look at the jinja PackageLoader. What's the advantage?
     with open(path, 'r') as f:
         return jinja2.Template(f.read())
 
 
+def load_model(path):
+    """
+    Loads a cellml model.
+    """
+    #TODO: This should become part of cellmlmanip
+
+    import cellmlmanip
+    import cellmlmanip.parser
+
+    model = cellmlmanip.parser.Parser(path).parse()
+    model.make_connections()
+    model.add_units_to_equations()
+    for e in model.equations:
+        model.check_left_right_units_equal(e)
+    return model
+
+
+def get_equations(graph):
+    """
+    Gets the equations from a cellml model graph.
+    """
+    #TODO: This should become part of cellmlmanip
+
+    # Get sorted symbols
+    #TODO: networkx is not a dependency
```

Comment:
As in, not listed as a dependency of `cellmlmanip`?

Reply:
It _is_ listed for cellmlmanip! The note is just to remind me that if we _don't_ move this code to cellmlmanip, I need to add networkx to fccodegen's dependencies